### DISCUSSION TITLE
[Q&A]  Using pre-trained initial model instead of training from scratch

### DISCUSSION THREAD

**Discussion by umgpy** [NVIDIA/NVFlare#2149] [Q&A]
### Python version (`python3 -V`)

3.9.12

### NVFlare version (`python3 -m pip list | grep "nvflare"`)

2.3.3

### Operating system

Ubuntu 20.04

### Have you successfully run any of the following examples?

- [ ] hello-numpy-sag with simulator
- [ ] hello-pt with simulator
- [ ] hello-numpy-sag with POC
- [ ] hello-pt with POC

### Please describe your question

I am following the nvflare-monai Prostate 3D example and I am trying to have an initial per-trained model, instead of training from scratch. Does one have to define that in the SupervisedLearner class?

**Comment by YuanTingHsieh** [comment]
@holgerroth or @ZiyueXu77 can you help on this when you got time, thanks!

**Comment by holgerroth** [comment]
For FedAvg, you can use the server to initialize the state of the global model before training. You can add an argument to the [PTFileModelPersistor](https://github.com/NVIDIA/NVFlare/blob/64666c966c751928b59be03f90cf3b64c9d9bc89/examples/advanced/prostate/prostate_3D/job_configs/prostate_fedavg/app/config/config_fed_server.json#L13C16-L13C36), namely [source_ckpt_file_full_name](https://github.com/NVIDIA/NVFlare/blob/64666c966c751928b59be03f90cf3b64c9d9bc89/nvflare/app_opt/pt/file_model_persistor.py#L41) which will load your pretrained checkpoint before sending the weights out to the clients for local training.
